spec/features/students_page_spec.rb

```diff
@@ -35,6 +35,10 @@ def js_visit(path)
                           real_name: 'Mr. Tester',
                           trained: true)
 
+    @user1 = create(:user, username: 'Allan.Aikins',
```

Comment:
This would be better if the variable reflected the way it's being used, like `@user_with_period`.

Reply:
Something like this? ` @user1 = create(:user, username: @user_with_period)` ?

Reply:
No, like `@user_with_period = create(...`